File: games/xracer/patches/patch-src_utils.c
$NetBSD: patch-src_utils.c,v 1.1 2013/06/27 15:49:43 joerg Exp $

--- src/utils.c.orig	2013-06-26 22:50:54.000000000 +0000
+++ src/utils.c
@@ -21,6 +21,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 
 #ifdef HAVE_UNISTD_H
 #include <unistd.h>

File: audio/spotifyd/patches/patch-src_utils.rs
$NetBSD: patch-src_utils.rs,v 1.2 2025/04/13 15:28:08 pin Exp $

Allow building on NetBSD.

--- src/utils.rs.orig	2025-04-13 15:09:04.578021815 +0000
+++ src/utils.rs
@@ -4,6 +4,7 @@ use std::{env, time::Duration};
 #[cfg(any(
     target_os = "freebsd",
     target_os = "linux",
+    target_os = "netbsd",
     target_os = "openbsd",
     target_os = "android"
 ))]
